Dockerfile.accumulo

```diff
@@ -0,0 +1,20 @@
+##########################################################
+# Copyright 2016 Crown Copyright, cybermaggedon
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################################################
+
+FROM cybermaggedon/accumulo:1.8.0
```

Comment:
There are currently some unresolved issues with Accumulo 1.8.0 and Gaffer (as well as a general lack of testing) - would it be possible to use Accumulo 1.7.2 for the moment?